projects/plugins/social/src/js/components/admin-page/test/index.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+import { useSelect, createReduxStore, register } from '@wordpress/data';
+import React from 'react';
+import { STORE_ID, storeConfig } from '../../../store';
+import Admin from '../index';
+
+const store = createReduxStore( STORE_ID, storeConfig );
+register( store );
```

Comment:
This is a bit ugly and probably worth exporting into a jest setup file. Alternatively, we can update the store init to be in line with other projects in monorepo: https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/connection/state/store.jsx#L19-L27

Reply:
> Alternatively, we can update the store init to be in line with other projects in monorepo

I think that's the way to go. Do you think we should do that first or jest get this merged?

Reply:
I'm fine merging it as is and then aligning the store registration in follow-up PR.